### ISSUE TITLE
ShaderPreprocessor incorrectly adds version to ES2 shaders.

### ISSUE THREAD

**Issue by sansumbrella** [cinder/Cinder#862]
Preprocessor is adding `#version 150` to ES2 shaders which prevents them from compiling on devices.

**Comment by ryanbartley** [comment]
That should've been fixed with #796

**Comment by sansumbrella** [comment]
Might be using a slightly older version, then. Just disabled the preprocessor and it works, but I can't use the convenience GlslProg constructors.

```
gl::GlslProg::Format fmt;
fmt.preprocess( false )
  .vertex( … )
  .fragment( … );
```
